src/containers/AuthenticationPage/SignupForm/SignupForm.js

```diff
@@ -74,13 +77,44 @@ const SignupFormComponent = props => (
         passwordMaxLength
       );
 
+      useEffect(() => {
+        setSelectedRole(values?.userType)
+
+      }, [values?.userType])
+
```

Comment:
Without using useEffect, the setSelectedRole will be called every time the form re-render, and it will cause errors. I only want the setSelectedRole execute when the userType changes, so I placed the setSelectedRole inside the useEffect, with the values?.userType as a dependency. So it is only called when the values?.userType changes.